src/app/interfaces/periodic-snapshot-task.interface.ts

```diff
@@ -1,7 +1,12 @@
 import { LifetimeUnit } from 'app/enums/lifetime-unit.enum';
 import { Schedule } from 'app/interfaces/schedule.interface';
 
-export interface PeriodicSnapshotTask {
+interface PeriodicSnapshotTaskState {
+  state: string;
+  error: string;
+}
+
+export interface PeriodicSnapshotTaskRaw {
```

Comment:
What does PeriodicSnapshotTaskRaw do? Doesn't look like it's used anywhere else?